cmd/lotus-bench/import.go

```diff
@@ -214,32 +214,14 @@ func countGasCosts(et *types.ExecutionTrace) (int64, int64) {
 	}
 
 	for _, sub := range et.Subcalls {
-		c, v := countGasCosts(&sub)
+		c, v := countGasCosts(&sub) //nolint
 		cgas += c
 		vgas += v
 	}
 
 	return cgas, vgas
 }
 
-func compStats(vals []float64) (float64, float64) {
-	var sum float64
-
-	for _, v := range vals {
-		sum += v
-	}
-
-	av := sum / float64(len(vals))
-
-	var varsum float64
-	for _, v := range vals {
-		delta := av - v
-		varsum += delta * delta
-	}
-
-	return av, math.Sqrt(varsum / float64(len(vals)))
-}
```

Comment:
@Kubuxu @whyrusleeping is this somehow useful?

Reply:
no, it is old code